RealmModelGenerator/EntitieViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  EntitieViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/21/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+
+class EntitiesViewController: NSViewController, EntitiesViewDelegate, EntitiesViewDataSource {
+    static let TAG = NSStringFromClass(EntitiesViewController)
+    
+    @IBOutlet weak var entitiesView: EntitiesView!
+    
+    private var schema:Schema = Schema()
+    private var model: Model = Schema.init().createModel()
+    private var entities:[Entity] = []
+    private var selectedIndex = -1
+    
+    override func viewWillAppear() {
+        entitiesView.delegate = self
+        entitiesView.dataSource = self
+        
+        if let currentModel = schema.getCurrentModel() {
+            model = currentModel
+        } else {
+            model = schema.createModel()
+        }
+
+        entities = model.entities
+    }
+    
+    override func viewDidLoad() {
```

Comment:
put viewDidLoad above viewDidAppear() {}
Its nice to have them in the lifecycle order they are called